sdk/nodejs/output.ts

```diff
@@ -520,7 +531,9 @@ export function containsUnknowns(value: any): boolean {
         else if (seen.has(val)) {
             return false;
         }
-        else if (val instanceof Array) {
+
+        seen.add(val);
```

Comment:
intent here is to remove the two 'adds' below now.

Reply:
Fixed.